content/docs/testing-farm.md

```diff
@@ -315,6 +315,27 @@ execute:
     how: tmt
 ```
 
+You can run rpminspect also using the CentOS Stream configuration. This should prepare you before opening CentOS Stream dist-git MRs.
+```yaml
+summary:
+    Fetch SRPM and check it with rpminspect
+discover:
+    how: shell
+    tests:
+      - name: rpminspect
+        test: rpminspect-centos -v -t VERIFY --profile=centos-stream-9-devel /tmp/*.src.rpm
+prepare:
+  - name: packages
+    how: install
+    package:
+    - rpminspect
+    - rpminspect-data-fedora
```

Comment:
I think you probably meant [rpminspect-data-centos](https://copr.fedorainfracloud.org/coprs/dcantrell/rpminspect/packages/).

Reply:
thank you for spotting this